plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/MySQLDialect.java

```diff
@@ -304,14 +305,30 @@ public String escapeScriptValue(DBSTypedObject attribute, @NotNull Object value,
     public boolean validIdentifierStart(char c) {
         return Character.isLetterOrDigit(c);
     }
-    
+
     @NotNull
     @Override
-    public String getTypeCastClause(@NotNull DBSTypedObject attribute, @NotNull String expression, boolean isInCondition) {
+    public String getTypeCastClause(
+        @NotNull DBSTypedObject attribute,
+        @NotNull String expression,
+        boolean isInCondition
+    ) {
```

Comment:
The previous declaration was shorter than 140 characters. Why wrap it?

Reply:
cause we have 120 symbols limit by default in idea project settings -_-